configs/access_log_format_helper_v3.template.yaml

```diff
@@ -0,0 +1,15 @@
+{% macro ingress_sampled_log() -%}
+  # log_format: "[%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH):256% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\"\n"
```

Comment:
for some reason i wasnt able to get any of the log_format strings to work - ill play with this a bit further...

Reply:
this seems to be fixed